hack/arktos-worker-up.sh

```diff
@@ -32,6 +35,8 @@ KUBELET_KUBECONFIG=${KUBELET_KUBECONFIG:-"${SECRET_FOLDER}/kubelet.kubeconfig"}
 KUBELET_CLIENTCA=${KUBELET_CLIENTCA:-"${SECRET_FOLDER}/client-ca.crt"}
 HOSTNAME_OVERRIDE=${HOSTNAME_OVERRIDE:-"$(hostname)"}
 CLUSTER_DNS=${CLUSTER_DNS:-"10.0.0.10"}
+FEATURE_GATES="AllAlpha=false,WorkloadInfoDefaulting=true,QPSDoubleGCController=true,QPSDoubleRSController=true,MandatoryArktosNetwork=true"
```

Comment:
Does MandatoryArktosNetwork needs to be true in the master host?
Currently arktos-up.sh and arktos-up-scale-out-poc.sh both have same feature gate setting: FEATURE_GATES="${FEATURE_GATES},WorkloadInfoDefaulting=true,QPSDoubleGCController=true,QPSDoubleRSController=true"
Can this be consolidating into hack/lib/common-var-init.sh?

Reply:
>> Does MandatoryArktosNetwork needs to be true in the master host?
yes. If it is not set true, the default DNS codes at https://github.com/CentaurusInfra/arktos/blob/master/pkg/kubelet/network/dns/dns.go#L102-L104 are used. But the codes for multi-tenant and multi-network at https://github.com/CentaurusInfra/arktos/blob/master/pkg/kubelet/network/dns/dns.go#L106-L149 are not used. 

>>Currently arktos-up.sh and arktos-up-scale-out-poc.sh both have same feature gate setting: >>FEATURE_GATES="${FEATURE_GATES},WorkloadInfoDefaulting=true,QPSDoubleGCController=true,QPSDoubleRSContr>>oller=true"
>>Can this be consolidating into hack/lib/common-var-init.sh?

Good suggestion. Let me research how to consolidate it. Thanks!

Reply:
new commit consolidates into hack/lib/common-var-init.sh